index.html

```diff
@@ -76,12 +88,8 @@ <h3 class="connect">LET'S CONNECT</h3>
         </div>
       </section>
     </div>
-
- <!-- MY WORK POTFOLIO SESSION-->
- 
     <section class="work" id="portfolio">
-      
-      <div class="card card-1">
+      <!-- <div class="card card-1">
```

Comment:
- [ ] Great job implementing the projects section dynamically. Now ensure that you delete all the commented out code from your code base which is making it look cluttered and messy.